src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -107,14 +150,42 @@ public IDisposable On(string methodName, Type[] parameterTypes, Func<object[], o
             return new Subscription(invocationHandler, invocationList);
         }
 
+        /// <summary>
+        /// Invokes a streaming hub method using the specified method name, return type and arguments.
+        /// </summary>
+        /// <param name="methodName">The hub method name.</param>
+        /// <param name="returnType">The hub return type.</param>
+        /// <param name="args">The hub arguments.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> to observe while waiting for the task to complete.</param>
+        /// <returns>
+        /// A <see cref="Task"/> that on completion indicates the hub method stream is open.
+        /// The value of the task is a <see cref="ChannelReader{T}"/> for the streamed hub method values.
+        /// </returns>
         public async Task<ChannelReader<object>> StreamAsChannelCoreAsync(string methodName, Type returnType, object[] args, CancellationToken cancellationToken = default) =>
             await StreamAsChannelCoreAsyncCore(methodName, returnType, args, cancellationToken).ForceAsync();
 
+        /// <summary>
+        /// Invokes a hub method using the specified method name, return type and arguments.
+        /// </summary>
+        /// <param name="methodName">The hub method name.</param>
+        /// <param name="returnType">The hub return type.</param>
+        /// <param name="args">The hub arguments.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> to observe while waiting for the task to complete.</param>
+        /// <returns>
+        /// A <see cref="Task"/> that on completion indicates the hub method invocation is complete.
+        /// The value of the task is a <see cref="object"/> for the hub method return value.
+        /// </returns>
         public async Task<object> InvokeCoreAsync(string methodName, Type returnType, object[] args, CancellationToken cancellationToken = default) =>
             await InvokeCoreAsyncCore(methodName, returnType, args, cancellationToken).ForceAsync();
 
-        // REVIEW: We don't generally use cancellation tokens when writing to a pipe because the asynchrony is only the result of backpressure.
-        // However, this would be the only "invocation" method _without_ a cancellation token... which is odd.
+        /// <summary>
+        /// Invokes to a hub method using the specified method name and arguments.
+        /// Does not wait for a response from the receiver.
+        /// </summary>
+        /// <param name="methodName">The hub method name.</param>
+        /// <param name="args">The hub arguments.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> to observe while waiting for the task to complete.</param>
+        /// <returns>A <see cref="Task"/> that on completion indicates the hub method send is complete.</returns>
```

Comment:
Depending on the Task blurb we choose, something like "indicates the invocation has been sent to the server" is a little clearer.